### DISCUSSION TITLE
V0.6.0 alpha-1

### DISCUSSION THREAD

**Discussion by DSheirer** [DSheirer/sdrtrunk#1454] [Announcements]
## 1. Features:
* SDRPlay RSP tuner support

## 2. Requirements:
* Windows x86 (64-bit)
* Linux x86/aarch64 (64-bit)
* OSX x86/aarch64 (64-bit)

**Note**: JMBE library must be manually downloaded/built for OSX aarch64 due to OSX sandboxing of downloaded code.
**Note**: Linux for arm-32 is NOT currently supported

## 3. Optional Requirements
**JMBE version 1.0.9+** is required for decoding APCO25 Phase 1/2 and DMR digital audio on Linux and OSX aarch64 and this library can be compiled within the sdrtrunk application.

Legacy instructions for compiling the JMBE library are located at: **https://github.com/DSheirer/sdrtrunk/wiki/JMBE**

## 4. User Manual
* [Wiki](https://github.com/DSheirer/sdrtrunk/wiki)
* [User Manual](https://github.com/DSheirer/sdrtrunk/wiki/User-Manual)

## 5. Support
* [Support](https://github.com/DSheirer/sdrtrunk/wiki/Support)

## 6. What's Changed
* #52 SDRPlay RSP tuner support. by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1453

**Full Changelog**: https://github.com/DSheirer/sdrtrunk/compare/v0.5.1...v0.6.0-alpha-1

<hr /><em>This discussion was created from the release <a href='https://github.com/DSheirer/sdrtrunk/releases/tag/v0.6.0-alpha-1'>V0.6.0 alpha-1</a>.</em>

**Comment by ghost** [comment]
Tried to run this, linux86 asset, and it doesn't start at all.

**Reply by DSheirer** [reply]
Well, that's not good.

**Reply by DSheirer** [reply]
> Tried to run this, linux86 asset, and it doesn't start at all.

Can you post either a screenshot or the error that you're seeing when you try to start the application?

**Comment by DSheirer** [comment]
This release is withdrawn.  Please use Version 0.6.0 alpha3
